tests/conftest.py

```diff
@@ -81,7 +81,13 @@
 except ImportError:
     _STREAMLIT_INSTALLED = False
 
-# All fixture functions have prefix 'fixture_' and explicitly declared name so they
+# Hypothesis settings
+from hypothesis import settings
+
+settings.register_profile("ci", deadline=2000, max_examples=200)
+settings.load_profile("ci")
```

Comment:
The documentation pipeline behaves weirdly here: According to the log, the problem with `is_mc` is still present, but the pipeline claims that the check passed. Also, it cannot find several type hint references which is fine here but is currently troublesome for me. Was there anything else changed that could be responsible for this?!?

Reply:
Sorry that was my bad